.gitignore

```diff
@@ -91,6 +91,7 @@ Doc/_minted-Tutorial/
 Doc/Tutorial.txt
 Doc/Tutorial.pdf
 Doc/Tutorial.html
+Doc/Tutorial.pyg
```

Comment:
What is a ``*.pyg`` file?

Reply:
When you build the documents for the Tutorial, it will create this file. It has the following contents: 

```
>>> from Bio.PDB import parse_pdb_header
>>> with open(filename, "r") as handle:
...     header_dict = parse_pdb_header(handle)
...

I don't where it comes from, it could also be that it comes from the doctests, but since it is the result of a test or build it belongs in the .gitignore.

Reply:
Ah - pygments, the code LaTeX minted uses for syntax colouring. Although I thought it used temp folders... never noticed that file before.